DuckDuckGo/NavigationBar/View/AddressBarTextField.swift

```diff
@@ -106,9 +106,8 @@ final class AddressBarTextField: NSTextField {
 
     private func subscribeToSelectedSuggestionViewModel() {
         selectedSuggestionViewModelCancellable = suggestionContainerViewModel?.$selectedSuggestionViewModel
-            .receive(on: DispatchQueue.main)
```

Comment:
Hey @mallexxx was `.receive(on: DispatchQueue.main)` causing the race condition as we were already on the main queue and it would dispatch in the next run loop?

Reply:
@mallexxx is this line removed to fix the actual bug or as an optimization? I'm asking because we're debugging a different issue together with @jotaemepereira (related to the search bar on New Tab Page), that seems to be caused by merging this PR, and when this line is re-added, the issue is gone. We'd like this to be brought back if possible, and just wanted to check how crucial to your PR it is. Thanks!